pkg/arvo/app/hark-store.hoon

```diff
@@ -380,17 +383,27 @@
       (~(put by archive-box) index notification(read %.y))
     (give %archive time index)
   ::
+  ::  if we detect cache inconsistencies, wipe and rebuild
```

Comment:
How frequently do these happen? Do we have any idea about root cause?

Reply:
When I first made the PR, I hadn't fixed the root cause of this inconsistency. So this was added as a temporary measure. I later found the root cause and fixed it in the second commit. The rebuild on inconsistency can be safely removed now, but I left in just in case we break it again down the line. Perhaps we should add a printf so we know whether or not this is occurring.

Reply:
Let's add that printf and keep it in here. Thanks for clarifying!